src/main/java/org/tron/common/runtime/utils/MUtil.java

```diff
@@ -1,40 +1,24 @@
 package org.tron.common.runtime.utils;
 
-import java.util.Arrays;
-import org.spongycastle.util.encoders.Hex;
-import org.tron.common.crypto.Hash;
 import org.tron.common.storage.Deposit;
 import org.tron.core.Wallet;
 import org.tron.core.actuator.TransferActuator;
 import org.tron.core.exception.ContractValidateException;
 
 public class MUtil {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make this class "final" or add a public constructor. ([squid:S2974](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2974))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1681&issues=AWbnEHSzSgN4uCcljsr8&open=AWbnEHSzSgN4uCcljsr8)

[ISSUE_KEY]: AWbnEHSzSgN4uCcljsr8